```

       > /build/d1hgm0pkhphps2f56blf4hfbjzdcjlwy-source/src/nvim/lua/treesitter.c: In function 'node_child_containing_descendant':
       > /build/d1hgm0pkhphps2f56blf4hfbjzdcjlwy-source/src/nvim/lua/treesitter.c:1095:18: error: invalid initializer
       >  1095 |   TSNode child = ts_node_child_containing_descendant(node, descendant);
       >       |                  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
``` 
I currently get this problem when trying to build with nix, so I couldn't verify that this works yet...

--- a/runtime/lua/vim/lsp/codelens.lua
+++ b/runtime/lua/vim/lsp/codelens.lua
@@ -164,7 +164,7 @@ function M.display(lenses, bufnr, client_id)
       return a.range.start.character < b.range.start.character
     end)
     for j, lens in ipairs(line_lenses) do
-      local text = lens.command and lens.command.title or 'Unresolved lens ...'
+      local text = (lens.command and lens.command.title or 'Unresolved lens ...'):gsub('%s+', ' ')
       table.insert(chunks, { text, 'LspCodeLens' })
       if j < num_line_lenses then
         table.insert(chunks, { ' | ', 'LspCodeLensSeparator' })

